ons_alpha/jinja2/component_overrides/footer/_macro.njk

```diff
@@ -0,0 +1,218 @@
+                                    onsList({
+                                        "classes": 'ons-u-mb-s ons-footer--rows',
+                                        "variants": ['bare', 'inline'],
+                                        "itemsList": legal.itemsList
+                                    })
+                                }}
+                            {% endfor %}
+                        {% endif %}
+
+                        {% if params.OGLLink %}
+                            <!-- OGL -->
+                            <div class="ons-footer__license footer__licence">
+                                {# hard-code the ogl icon in order to override the inline fill to set the colour #}
+                                <svg
+                                    class="ons-footer__ogl-img footer__ogl-img{{ iconClasses }}"
+                                    xmlns="http://www.w3.org/2000/svg"
+                                    width="50px"
+                                    height="20px"
+                                    viewBox="0 0 60 24"
+                                    focusable="false"
+                                    aria-hidden="true"
+                                    role="img"
+                                >
+                                    <title>{{ params.altText | default ('Open Government License logo') }}</title>
+                                    <path
+                                        d="M51.7,17.5V0l-6.2,4v19.8h13.8v-6.2H51.7z M36.7,16.3c-1,0.9-2.4,1.4-3.8,1.4c-3.2,0-5.8-2.6-5.8-5.8s2.6-5.8,5.8-5.8c2,0,3.9,1.1,4.9,2.7L43,5.6C40.9,2.2,37.1,0,32.9,0c-4.5,0-8.4,2.5-10.4,6.1C20.4,2.5,16.5,0,12,0C5.4,0,0,5.4,0,12s5.4,12,12,12c4.5,0,8.4-2.5,10.4-6.1c2.1,3.6,6,6.1,10.4,6.1c3,0,5.8-1.1,7.9-3l2.4,2.7h0.4V13h-9.8L36.7,16.3zM12,17.8c-3.2,0-5.8-2.6-5.8-5.8S8.8,6.2,12,6.2s5.8,2.6,5.8,5.8S15.2,17.8,12,17.8"
+                                        fill="#F5F5F6"
+                                    ></path>
+                                </svg>
+                                {% if params.OGLLink.HTML %}
+                                    {{ params.OGLLink.HTML | safe }}
+                                {% elif params.OGLLink %}
+                                    <div>
+                                        {% from "components/external-link/_macro.njk" import onsExternalLink %}
+                                        {% if params.lang == 'cy' %}
+                                            {{ params.OGLLink.pre | default('Mae’r holl gynnwys ar gael o dan y') }}
+                                            {{
+                                                onsExternalLink({
+                                                    "url": params.OGLLink.url | default('https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/'),
+                                                    "linkText": params.OGLLink.link | default('Drwydded Llywodraeth Leol v3.0'),
```

Comment:
This code looks quite odd - is it jinja specific? Does the following not work?

```suggestion
                                                    "url": params.OGLLink.url 
or 'https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/',
                                                    "linkText": params.OGLLink.link or 'Drwydded Llywodraeth Leol v3.0',
```

Reply:
And ditto...